e2e/test/scenarios/dashboard-cards/dashboard-card-resizing.cy.spec.js

```diff
@@ -292,7 +295,10 @@ describe("scenarios > dashboard card resizing", () => {
           editDashboard();
 
           dashcards.forEach(({ card }) => {
-            const dashcard = cy.contains(".DashCard", card.name);
+            const dashcard = cy.contains(
+              "[data-testid='dashcard-container']",
```

Comment:
nit: To make this the same as the above change.
```suggestion
              "[data-testid=dashcard-container]",
```